src/Language/Haskell/LSP/Core.hs

```diff
@@ -638,10 +698,6 @@ initializeRequestHandler' (_configHandler,dispatcherProc) mHandler tvarCtx req@(
     atomically $ modifyTVar' tvarCtx (\c -> c { resWorkspaceFolders = wfs })
 
     ctx0 <- readTVarIO tvarCtx
-
-    -- capture initialize request
-    captureFromClient (ReqInitialize req) (resCaptureFile ctx0)
```

Comment:
Why has this been removed?

Reply:
It's already captured by the wrapper to this function. Is there a problem in capturing twice? I'm proceeding from the understanding that it is a problem indeed